src/components/MultipleBeneficiariesAmount.tsx

```
@@ -0,0 +1,293 @@
+							{parseBalance(
+								isGenralIndexExist
+									? requestedAmountFormatted?.mul(new BN(Number(currentTokenPrice) * 10 ** chainProperties?.[network]?.tokenDecimals))?.toString() || '0'
+									: totalAmountInChainSymbol?.toString(),
+								0,
+								!isGenralIndexExist,
+								network
+							)}
+							<InfoCircleOutlined className={classNames(theme == 'dark' ? 'text-icon-dark-inactive' : 'text-bodyBlue')} />
+						</div>
+					}
+				</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)